waku/v2/node/rest/store/handlers.nim

```diff
@@ -0,0 +1,251 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  chronicles,
+  json_serialization,
+  presto/route
+import
+  ../../../../common/base64,
+  ../../../protocol/waku_message/topics/content_topic,
+  ../../../protocol/waku_store/common,
+  ../../../utils/time,
+  ../../waku_node,
+  ../../peer_manager,
+  ../responses,
+  ../serdes,
+  ./types
+
+export types
+
+logScope:
+  topics = "waku node rest store_api"
+
+const futTimeout* = 5.seconds # Max time to wait for futures
+
+
+# Adds a peer-store-node to the peer manager
+# @params:
+#   storeAdd: fully qualified peer multiaddr, in the
+#             format `(ip4|ip6)/tcp/p2p/$peerId` but URL-encoded
+#   node: WakuNode whose peerManager will get the new store-node
+proc setStorePeer(storeAddr: string,
+                  node: WakuNode):
+                  Option[RestApiResponse] =
+
+  let parsedAddr = decodeUrl(storeAddr)
+  debug "Setting store node ", parsedAddr
+
+  var storenode: RemotePeerInfo = nil
+  try:
+    storenode = parseRemotePeerInfo(parsedAddr)
+  except Exception as e:
+    error "Failed parsing remote peer info", error=e.msg
+    return some(RestApiResponse.badRequest(e.msg))
+
+  try:
+    node.peerManager.addServicePeer(storenode, WakuStoreCodec)
```

Comment:
note that this will override the existing service peer configure in the CLI, as we only have one set for each protocol. not sure if intended, just fyi :)

Reply:
Thanks @alrevuelta !
@LNSD recommended me to simplify this. We will no longer deal with peerManager. Therefore, this part should be simplified and directly make a request throughout the already configured wakuStoreClient instance, and not add any 'WakuStoreCodec' at this point.